python/cudf/cudf/core/udf/pipeline.py

```diff
@@ -220,22 +287,41 @@ def _kernel(retval, input_col_0, input_col_1, offset_0, offset_1, size):
         row_initializers.append(row_initializer)
 
     masked_input_initializers = "\n".join(initializers)
-    row_initializers = "\n".join(row_initializers)
+    row_initializer_list = "\n".join(row_initializers)
 
     # Incorporate all of the above into the kernel code template
     d = {
         "input_columns": input_columns,
         "input_offsets": input_offsets,
         "masked_input_initializers": masked_input_initializers,
-        "row_initializers": row_initializers,
+        "row_initializers": row_initializer_list,
```

Comment:
I saw this was tweaked to "pacify mypy." 😉 What if you did the join call here? It would eliminate the awkward intermediate variable. The intermediate name doesn't really help with descriptions because it's not a list, it's a string. 😄
```suggestion
        "row_initializers": "\n".join(row_initializers),
```
I support the same thing for `masked_input_initializers`, for symmetry.